lib/zip/extra_field.rb

```diff
@@ -83,8 +83,8 @@ def local_size
       to_local_bin.bytesize
     end
 
-    alias :length :local_size
-    alias :size :local_size
+    alias_method :length, :local_size
+    alias_method :size, :local_size
```

Comment:
There is no sense to use alias_method because we don't need chains. So better use alias instead of alias_method